public/locally-available-studies.json

```diff
@@ -3,29 +3,29 @@
     {
       "name": "Ion Base Study",
       "icons": {
-        "32": "https://addons.cdn.mozilla.net/user-media/addon_icons/2644/2644632-32.png?modified=4a64e2bc",
-        "64": "https://addons.cdn.mozilla.net/user-media/addon_icons/2644/2644632-64.png?modified=4a64e2bc",
-        "128": "https://addons.cdn.mozilla.net/user-media/addon_icons/2644/2644632-128.png?modified=4a64e2bc"
+        "32": "https://addons.cdn.mozilla.net/static/img/addon-icons/default-32.png",
+        "64": "https://addons.cdn.mozilla.net/static/img/addon-icons/default-64.png",
+        "128": "https://addons.cdn.mozilla.net/static/img/addon-icons/default-128.png"
       },
       "schema": 1597266497978,
       "authors": {
-        "url": "https://addons.mozilla.org/en-US/firefox/addon/pioneer-v2-example/",
-        "name": "Pioneer Developers"
+        "url": "https://addons.mozilla.org/en-US/firefox/addon/ion-basic-study/",
+        "name": "Ion Developers"
       },
-      "version": "1.0",
-      "addon_id": "pioneer-v2-example@mozilla.org",
+      "version": "2.0",
+      "addon_id": "ion-basic-study@mozilla.org",
       "moreInfo": {
-        "spec": "https://addons.mozilla.org/en-US/firefox/addon/pioneer-v2-example/"
+        "spec": "https://addons.mozilla.org/en-US/firefox/addon/ion-basic-study/"
       },
       "isDefault": false,
       "sourceURI": {
-        "spec": "/public/ion-base-addon.xpi"
+        "spec": "https://addons.mozilla.org/firefox/downloads/file/3659906/ion_basic_study-2.0-fx.xpi"
```

Comment:
This and the version number need to change whenever we update the study add-on on AMO. One benefit here though is that we can point it at pre-release versions and test that way before making add-on updates live.